build/test-requirements.txt

```diff
@@ -1,6 +1,6 @@
 flake8
 # For now, we pin the numpy version here
-numpy>=1.16,<1.20
+numpy>=1.16
```

Comment:
Unfortunately you will need to leave this pin here, because the released jaxlib has a < 1.20 constraint. I have removed it but I only noticed after the build was complete.

Reply:
Strangely the CI actually passes. Somehow the package installation doesn't notice the conflict. However, it does show up if I install locally with pip.

Reply:
So pip will install 1.20 and then it will just abort when trying to install jaxlib? I will change and leave a comment.

Reply:
I left <1.20 in and I added a comment in CHANGELOG.md in a place that I thought the next person to release jaxlib will see. That is when we should relax the test-requirements pin.